scripts/checkout_submodules.py

```diff
@@ -109,6 +117,7 @@ def main():
     logging.basicConfig(format='%(message)s', level=logging.INFO)
 
     parser = argparse.ArgumentParser(description='Checkout or update relevant git submodules')
+    parser.add_argument('--configure', action='store_true', help='Configure global git options if necessary, e.g. safe.directory')
```

Comment:
`--configure` is not obvious if read outside of context/help (i.e. configure what?)

This seems to specifically only affect chip-root safe directory. Making it a "generic configure git" seems unnecessary, so I would apply a "You aren't gonna need it". Lets name it `--ensure-root-is-git-safe` or similar.

Reply:
My intent was to capture something like "it's OK to modify the global configuration"... `--ensure-root-is-git-safe` sounds kinda innocuous... who wouldn't want to ensure things are safe.

Another naming option I thought of was `--ci`, as in "do things that are valid in the context of CI". At least that makes it somewhat clear that one shouldn't blindly add this option for every call to the script.

Something like `--allow-git-global-config` would be the most explicit I guess, if rather clunky.

Reply:
I'm going to go with `--allow-changing-global-git-config`